pkg/probe/OWNERS

```diff
@@ -1,11 +1,9 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - Random-Liu
-  - dchen1107
-  - derekwaynecarr
-  - tallclair
-  - vishh
-  - yujuhong
+  - sig-node-approvers
```

Comment:
this removes @tallclair from the list. Intentional?

Reply:
https://github.com/kubernetes/kubernetes/commit/933a2ca634cc71b77e907558df68920f7c143629

```
 # see https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES#LC220:~:text=sig%2Dnode%2Dreviewers
# see https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES#LC220:~:text=sig%2Dnode%2Dapprovers
```

The comments help when I read it the first time.

Reply:
/hold

holding to make sure that removing one approver is intentional. Adding more like @mrunalp who cannot approve this PR is definitely an improvement. Feel free to un-hold

Reply:
I'll quickly ask @tallclair, I don't think he's been active in this code but he's the only effective removal.

Reply:
/hold cancel

He said he's fine being removed :)